hw/ip/pattgen/data/pattgen.hjson

```diff
@@ -42,12 +42,18 @@
         { bits: "0",
           resval: "0",
           name: "ENABLE_CH0",
-          desc: "Enable pattern generator functionality for Channel 0"
+          desc: "Enable pattern generator functionality for Channel 0",
+          tags: [// don't write (e.g. 1'b1) to enable bit
+                 // because DUT can operate with zero values programmed to registers - it affects several other csrs
+                 "excl:CsrExclWrite:CsrExclWriteCheck"]
         }
         { bits: "1",
           resval: "0",
           name: "ENABLE_CH1",
-          desc: "Enable pattern generator functionality for Channel 1"
+          desc: "Enable pattern generator functionality for Channel 1",
+          tags: [// don't write (e.g. 1'b1) to enable bit
+                 // because DUT can operate with zero values programmed to registers - it affects several other csrs
+                 "excl:CsrExclWrite:CsrExclWriteCheck"]
```

Comment:
Fixed

Reply:
I update the tags but could you explain the usage of `CsrNonInitTests`? CsrExclWriteCheck should be removed because it is already covered by `CsrExclWrite`

Reply:
this tags has 3 columns
1. excl: for exclusion
2. what test to exclude, we usually have 2 options here, CsrAllTests or CsrNonInitTests. The first one is to apply for all the test, the 2nd one is to apply for all except csr_hw_reset, since we can write anything to the reg, then do reset and read back to check default value. I guess this reg can be written with some random value before reset, so use `excl:CsrNonInitTests:CsrExclWrite`. In some case, we should never write it, then use `excl:CsrAllTests:CsrExclWrite`
3. what action to exclude, write, check or all.

Reply:
For `enable ` bits, you can write `0` but not write `1`